inst/examples/03-formats.Rmd

```diff
@@ -218,9 +231,210 @@ favicon: "favicon.ico"
+
+In addition to the basic **bookdown** components (Section \@ref(components)), the main features of `bs4_book` are:
+
+- Easy customisation of colours and fonts with
+  the **bslib** package (<https://pkgs.rstudio.com/bslib/>)
+
+- Built-in search (broken down by section) that helps readers quickly find what
+  they are looking for.
+
+- A sidebar containing a within-chapter table of contents that makes
+  navigation easy and helps provide context about your current position
+  within the chapter.
+
+- Thoughtful typography to make the contents as easy as possible to read,
+  regardless of the size of your device. A sticky header gets out of your
+  way when reading, but is easily accessible if you need it.
+
+- In-line footnotes mean you can read asides next to the next they refer
+  to. This theme is best paired with a reference style that generates
+  footnotes.
+
+- R syntax highlighting and autolinking by
+  the **downlit** package (<https://downlit.r-lib.org/>) is paired with a accessible
+  colour scheme designed by Alison Hill.
+  
+- Enhanced metadata for social sharing via platforms like Twitter, LinkedIn, and Facebook, so that each chapter shared will have a unique description, auto-generated based on that chapter's content.
+
+- Ability to configure links to a remote repository like GitHub or GitLab, allowing readers to easily view each chapter's source file or suggest an edit.
+
+The output format function in **bookdown** is `bs4_book()`. Here are its arguments:
+
+```{r bs4-book-args, code=formatR::usage(bookdown::bs4_book, output=FALSE, fail='none'), eval=FALSE, R.options=list(width=50)}
+```
+
+This format also requires extra dependencies that you'll be prompted to install: `r knitr::combine_words(bookdown:::bs4_book_deps())`.
+
+#### Writing a `bs4_book`
+
+The easiest way to get started writing a new `bs4_book` is to use the RStudio Project Wizard (*File > New Project > New Directory > Book project using bookdown*) and select `bs4_book` from the dropdown menu: 
+
+```{r fig.align='center', echo=FALSE, include=identical(knitr:::pandoc_to(), 'html'), fig.alt="Screenshot of the RStudio Project Wizard for creating a new bs4_book project."}
+knitr::include_graphics('images/new-bs4-book.png', dpi = NA)
+```
+
+If you don't use RStudio or prefer a function, you can create the same project template with `bookdown::create_bs4_book()` from your R console. See `?create_bs4_book` for help.
+
+This theme is designed for books that use one chapter per page. 
+This means that each chapter is an `.Rmd` file, and each `.Rmd` file can contain one (and only one) chapter. 
+A chapter *must* start with a first-level heading: `# A good chapter`, and can contain one (and only one) first-level heading. 
+
+Use second-level and higher headings within chapters like: 
```

Comment:
Lower or higher? I often say "lower" because they are contained inside upper-level headings. I can understand that "higher" also makes sense because the numbers are higher (first, second, third...).

```suggestion
Use second-level and lower-level headings within chapters like: 
```